tree/30_generic_methods/logger_rudder.cf

```diff
@@ -33,14 +33,14 @@ bundle agent logger_rudder(message, class_prefix)
       "expected_reports_temp"   string => "${expected_reports_source}.tmp";
       "expected_reports_file"   string => "${expected_reports_source}.res";
 
-    logger_rudder_temp_resfile_repaired::
+    (!final_resfile_exists.logger_rudder_temp_resfile_reached|logger_rudder_temp_resfile_repaired)::
       # 2/ If the temporary file has been updated, read it to get all values in an array, and canonify the first entry
       "dim" int => readstringarrayidx("reports", "${expected_reports_temp}", "\s*#[^\n]*", ";;", 9999, 999999);
       "keys" slist => getindices("reports");
 
       "reports[${keys}][canon]" string => canonify("${reports[${keys}][1]}");
 
-    (logger_rudder_final_resfile_repaired|logger_rudder_temp_resfile_kept)::
+    (logger_rudder_final_resfile_repaired|logger_rudder_temp_resfile_kept.!logger_rudder_temp_resfile_repaired.final_resfile_exists)::
       "c_class_prefix" string => canonify("${class_prefix}");
```

Comment:
Er, you merged but I hadn't actually change the code. I was just asking your opinion. 
-- 
Sent from Kaiten Mail. Please excuse my brevity.